projects/igniteui-angular/src/lib/grids/grid/grid-filtering-ui.spec.ts

```diff
@@ -2794,6 +2703,131 @@ describe('IgxGrid - Filtering Row UI actions #grid', () => {
             expect(grid.rowList.length).toEqual(1);
         }));
     });
+
+    describe('Filtering events', () => {
+        let fix: ComponentFixture<any>;
+        let grid: IgxGridComponent;
+
+        const cal = SampleTestData.timeGenerator;
+        const today = SampleTestData.today;
+
+        beforeEach(fakeAsync(() => {
+            fix = TestBed.createComponent(IgxGridDatesFilteringComponent);
+            fix.detectChanges();
+            grid = fix.componentInstance.grid;
+
+            spyOn(grid.filtering, 'emit');
+            spyOn(grid.filteringDone, 'emit');
+        }));
+
+        it('Should emit filteringDone when we clicked reset - Date column type', fakeAsync(() => {
+            emitFilteringDoneOnResetClick(
+                fix,
+                grid,
+                cal.timedelta(today, 'day', 15),
+                'ReleaseDate',
+                IgxDateFilteringOperand.instance().condition('equals')
+            );
+        }));
+
+        it('Should emit filteringDone when we clicked reset - String column type', fakeAsync(() => {
+            emitFilteringDoneOnResetClick(
+                fix,
+                grid,
+                'search',
+                'ProductName',
+                IgxStringFilteringOperand.instance().condition('contains')
+            );
+        }));
+
+        it('Should emit filteringDone when we clicked reset - Number column type', fakeAsync(() => {
```

Comment:
Can we expand these tests to Boolean and Currency columns (not sure if we have some other type as well - Time probably?)?

Reply:
Yes, we can. I will add more columns in grid sample data with the (boolean, currency and time) column types.